examples/rpmsgsocket/rpsock_client.c

```diff
@@ -225,14 +228,14 @@ static int rpsock_unsync_test(struct rpsock_arg_s *args)
   return 0;
 }
 
-static int rpsock_stream_client(int argc, char *argv[])
+static int rpsock_stream_client(FAR char *argv[])
```

Comment:
no need for `argc`? just an example code?

Reply:
Thank you, I think it has already been determined in the main and should no longer be needed here